pkg/azurediskplugin/main.go

```diff
@@ -44,39 +44,42 @@ var (
+	getNodeInfoFromLabels                    = flag.Bool("get-node-info-from-labels", consts.DefaultGetNodeInfoFromLabels, "boolean flag to get zone info from node labels in NodeGetInfo")
+	disableAVSetNodes                        = flag.Bool("disable-avset-nodes", consts.DefaultDisableAVSetNodes, "disable DisableAvailabilitySetNodes in cloud config for controller")
+	vmType                                   = flag.String("vm-type", consts.DefaultVMType, "type of agent node. available values: vmss, standard")
+	enablePerfOptimization                   = flag.Bool("enable-perf-optimization", consts.DefaultEnablePerfOptimization, "boolean flag to enable disk perf optimization")
+	cloudConfigSecretName                    = flag.String("cloud-config-secret-name", consts.DefaultCloudConfigSecretName, "cloud config secret name")
+	cloudConfigSecretNamespace               = flag.String("cloud-config-secret-namespace", consts.DefaultCloudConfigSecretNamespace, "cloud config secret namespace")
+	customUserAgent                          = flag.String("custom-user-agent", consts.DefaultCustomUserAgent, "custom userAgent")
+	userAgentSuffix                          = flag.String("user-agent-suffix", consts.DefaultUserAgentSuffix, "userAgent suffix")
+	useCSIProxyGAInterface                   = flag.Bool("use-csiproxy-ga-interface", consts.DefaultUseCSIProxyGAInterface, "boolean flag to enable csi-proxy GA interface on Windows")
+	enableDiskOnlineResize                   = flag.Bool("enable-disk-online-resize", consts.DefaultEnableDiskOnlineResize, "boolean flag to enable disk online resize")
+	allowEmptyCloudConfig                    = flag.Bool("allow-empty-cloud-config", consts.DefaultAllowEmptyCloudConfig, "Whether allow running driver without cloud config")
+	enableAsyncAttach                        = flag.Bool("enable-async-attach", consts.DefaultEnableAsyncAttach, "boolean flag to enable async attach")
+	enableListVolumes                        = flag.Bool("enable-list-volumes", consts.DefaultEnableListVolumes, "boolean flag to enable ListVolumes on controller")
+	enableListSnapshots                      = flag.Bool("enable-list-snapshots", consts.DefaultEnableListSnapshots, "boolean flag to enable ListSnapshots on controller")
+	enableDiskCapacityCheck                  = flag.Bool("enable-disk-capacity-check", consts.DefaultEnableDiskCapacityCheck, "boolean flag to enable volume capacity check in CreateVolume")
+	kubeClientQPS                            = flag.Int("kube-client-qps", consts.DefaultKubeClientQPS, "QPS for the rest client. Defaults to 15.")
+	vmssCacheTTLInSeconds                    = flag.Int64("vmss-cache-ttl-seconds", consts.DefaultVMSSCacheTTLInSeconds, "vmss cache TTL in seconds (600 by default)")
+	enableAzureClientAttachDetachRateLimiter = flag.Bool("enable-attach-detach-rate-limiter", consts.DefaultEnableAzureClientAttachDetachRateLimiter, "azure client rate limiter for attach/detach operations (true by default)")
```

Comment:
The command-line parameters have been deprecated, though we still use them in the AKS RP. I am OK leaving these in for now, but we should add them to the list of deprecated parameters [here](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/cee1ff90d97c3776fd140b1491b4d04aad9a4dea/pkg/azureconstants/azure_constants.go#L207).

Reply:
I see now that you have already added them.